examples/pylab_examples/figlegend_demo.py

```diff
@@ -1,19 +1,19 @@
 import numpy as np
```

Comment:
While you are at it, do you mind adding a docstring to give this example a title and explain what it does? See http://matplotlib.org/devdocs/devel/MEP/MEP12.html for examples.